docker/tensorflow-application/video-serving/streaming.py

```diff
@@ -0,0 +1,61 @@
+from flask import Flask, Response, request
```

Comment:
Where is this being run from? I don't see a Dockerfile using this. My suggestion would be to split the `docker/tensorflow-application/analysis/Dockerfile` file into a multi-stage build file with the base image being everything common between the analysis and streaming images and then a separate stage for each one so the images can be built separately. Also, there's no indication whether they need to be run in the same pod or not.

Reply:
Still have a question about this. It needs it's own Dockerfile so it can be turned into an image. Maybe that's coming in a future commit?

Reply:
Yup, you're right. Currently, I was testing it by running this script on my local and couldn't experiment with a Dockerfile. That is definitely the next step here.